README.md

```diff
@@ -1 +1,43 @@
-# Microverse-Capstone-Concert
\ No newline at end of file
+# Microverse-Capstone-Concert
+
+> This is a Microverse capstone project to design a concert website template from behance.  
+
+## Built With
+
+- HTML 
+- CSS
+- Javascript
+
+## Link to Online Version
+
+- https://enning94.github.io/Concert-Website-Microverse-Capstone/
+
+## Link to Video presentation of website
```

Comment:
- [ ] @Enning94 great presentation, I loved it, although you forgot to describe what the project is about, in your case  I can see it is a music festival, kindly add a video describing your project.